### DISCUSSION TITLE
Does anyone have the legacy chartciulator .pbiviz file?

### DISCUSSION THREAD

**Discussion by varundoi** [zBritva/charticulator-visual-community#28] [General]
My PBI-Charticulator visuals have stopped working and I am unable to export the template since the legacy Charticulator visual has been removed from App source. Can someone share a link similar to this (https://github.com/deldersveld/power-bi-sunset-visuals/tree/master/Custom-visuals) to obtain the custom visual. This will help in exporting the template and migrate to the community edition. Thanks!

**Comment by varundoi** [comment]
Just FYI: This helped. https://www.reddit.com/r/PowerBI/comments/1ff1ov8/charticulator_bar_chart_disappeared/

**Comment by zBritva** [comment]
FYI: https://www.linkedin.com/posts/chitrachedu-raghava-59754023b_powerbi-charticulator-dataviz-activity-7244044623587737603-uqr3?utm_source=share&utm_medium=member_desktop

**Comment by zBritva** [comment]
FYI: https://community.fabric.microsoft.com/t5/Desktop/Power-BI-Charticulator-Visual-gone/td-p/4147430
